lib/withdrawal_tokens/cubit/confirm_withdrawal_cubit.dart

```diff
@@ -59,13 +60,14 @@ class ConfirmWithdrawalCubit extends Cubit<ConfirmWithdrawalState> {
         customFee: customFee,
       );
       logger.i(
-        'before execute: withdrawal from secretKey: ${sourceKeystore.secretKey} '
-        ', publicKey: ${sourceKeystore.publicKey} '
+        'before execute: withdrawal from secretKey: ${sourceKeystore.secretKey}'
+        ' , publicKey: ${sourceKeystore.publicKey} '
         'amount: $amount '
         'networkFee: $customFee '
-        'address: ${sourceKeystore.address} =>To address: ${state.withdrawalAddress}',
+        'address: ${sourceKeystore.address} =>To address: '
+        '${state.withdrawalAddress}',
       );
-      operationsList.executeAndMonitor();
+      await operationsList.executeAndMonitor();
```

Comment:
@bibash28 No need to await here. It will be done in the background, it takes some minutes to finish.

Reply:
ok